src/components/Calculator.jsx

```diff
@@ -1,34 +1,49 @@
-import './Calculator.css';
 import React from 'react';
+import './Calculator.css';
+import calculate from '../logic/calculate';
+
 
 class Calculator extends React.Component {
     constructor(props) {
         super(props);
+        this.state = { calcObj: {}}
+        this.handleClick = this.handleClick.bind(this);
+    }
+    handleClick(e){
+        const newCalcObj = calculate(this.state.calcObj, e.target.innerHTML )
+        this.setState({calcObj: {...newCalcObj}})
+    }
+    componentDidMount() {
+        this.setState( prevState => ({
+            calcObj :{
+                total: 0
+            }
+        }));
     }
     render(){
         return (
             <>
-                <span role="textbox"contentEditable className='calc-input'> </span>
+                <span role="textbox"contentEditable className='calc-input' suppressContentEditableWarning={true}> {this.state.calcObj.total}</span>
```

Comment:
- [ ] [Optional]Your project shows quite a good understanding of React basics, excellent work :clap:  however it will be nice if the user can see his operations as he enters number inputs instead of seeing just the result displayed